CMakeLists.txt

```diff
@@ -191,6 +191,11 @@ if(ADDRESS_SANITIZER)
 endif()
 
 #------------------------------------------------------------------------------
+# Enable usage statistics
+#------------------------------------------------------------------------------
+option(ENABLE_USAGE_STATISTICS "Enable collection of usage statistics" ON)
```

Comment:
could you print a status message during cmake configure, such that the user is made aware
of this please.
Similar to the way we do it for GIT_TRACKING